src/VisualStudio/Core/Test/AnalyzerSupport/AnalyzerDependencyCheckerTests.vb

```diff
@@ -15,26 +16,6 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests
     Public Class AnalyzerDependencyCheckerTests
         Inherits TestBase
 
-        Private Shared s_msbuildDirectory As String
-        Private Shared ReadOnly Property MSBuildDirectory As String
-            Get
-                If s_msbuildDirectory Is Nothing Then
-                    Dim vsVersion = If(Environment.GetEnvironmentVariable("VisualStudioVersion"), "14.0")
-                    Dim key = Registry.LocalMachine.OpenSubKey(Invariant($"SOFTWARE\Microsoft\MSBuild\ToolsVersions\{vsVersion}"), False)
-
-                    If key IsNot Nothing Then
-                        Dim toolsPath = key.GetValue("MSBuildToolsPath")
-                        If toolsPath IsNot Nothing Then
-                            s_msbuildDirectory = toolsPath.ToString()
-                        End If
-                    End If
-                End If
-
-                Return s_msbuildDirectory
-            End Get
-        End Property
-
-        Private Shared s_CSharpCompilerExecutable As String = If(MSBuildDirectory IsNot Nothing, Path.Combine(MSBuildDirectory, "csc.exe"), Nothing)
```

Comment:
deadcode?

Reply:
The test was directly invoking csc.exe as a way to build test assemblies. Since that stopped working all tests were skipped. I unskipped them and just use Roslyn compilations directly which is a lot easier in the first place.